docs/writing-rules/data-flow/taint-mode/overview.md

```diff
@@ -0,0 +1,300 @@
+---
+slug: overview
+title: Taint analysis
+hide_title: true
+description: Learn about taint mode, which allows you to write rules that catch complex injection bugs using taint analysis.
+tags:
+ - Rule writing
+ - Dataflow analysis
+ - Taint analysis
+---
+
+# Taint analysis overview
+
+Semgrep supports [taint analysis](https://en.wikipedia.org/wiki/Taint_checking), also known as taint tracking, through taint rules. Taint rules are specified by the inclusion of `mode: taint` in your rule.
```

Comment:
Maybe we can cross-link to and from our learning guides: https://deploy-preview-2346--semgrep-docs-prod.netlify.app/docs/writing-rules/data-flow/taint-mode/overview